### PR TITLE
Did quite a lot

### PR DESCRIPTION
**Pull request by wolfieboy09**

### CODE REVIEW

File: src/main/java/net/thejeezed/craftplusplus/block/crops/CottonCropBlock.java
Comment (AvacadoWizard120) [inline src/main/java/net/thejeezed/craftplusplus/block/crops/CottonCropBlock.java:1]: Don't really want you modifying this until it's finished.

File: src/main/java/net/thejeezed/craftplusplus/datagen/ModBlockStateProvider.java
Comment (AvacadoWizard120) [inline src/main/java/net/thejeezed/craftplusplus/datagen/ModBlockStateProvider.java:1]: Don't really want you modifying this until it's finished either.

File: src/main/java/net/thejeezed/craftplusplus/util/ItemUtils.java
Comment (AvacadoWizard120) [inline src/main/java/net/thejeezed/craftplusplus/util/ItemUtils.java:1]: looks good

**Comment by wolfieboy09** [comment]
Yay

**Review by AvacadoWizard120 — CHANGES_REQUESTED** [review]
Don't touch the CottonCropBlock or the ModBlockStateProvider for now, please.

**Comment by luhcartimods** [comment]
Don't touch my precious datagen

**Comment by wolfieboy09** [comment]
You need to tell me this before ;-;

**Comment by AvacadoWizard120** [comment]
> You need to tell me this before ;-;

lmao ur good bro, just revert the changes from those classes and submit another pull request so we can merge.

**Pull request closed without merging**